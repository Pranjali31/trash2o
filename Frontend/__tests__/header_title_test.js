import React from 'react';
import { render, fireEvent, waitFor } from '@testing-library/react-native';
import { NavigationContainer } from '@react-navigation/native';
import HeaderTitle from '../src/components/atoms/headerTitle/index';

test("index Component Loading", () => {
    render(<HeaderTitle />)

})
test("index component snapshot", () => {
    let component = render(<HeaderTitle/>).toJSON();
    expect(component).toMatchSnapshot();
})

describe("index Testing component", () => {
    var component;
    beforeEach(() => {
        component = render(<HeaderTitle />).toJSON();;
        //console.log(JSON.stringify(component))

    })
    
    test("Testing static text data", () => {
        let styles = component.children[0].children[0];
        expect(styles).toStrictEqual('Tras.h2o');
    })
})