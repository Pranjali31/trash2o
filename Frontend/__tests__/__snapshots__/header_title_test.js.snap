// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index component snapshot 1`] = `
<View>
  <Text>
    Tras.h2o
  </Text>
</View>
`;
